README.md

```diff
@@ -112,6 +114,23 @@ Canvas Kit follows [semantic versioning](https://semver.org/) and is enforced au
 
 Each module is independently versioned using [Lerna](https://github.com/lerna/lerna).
 
+## Version Support
+
+At any given time, we support three major versions of Canvas Kit: previous, current, and next. Each
+of these have different levels of support.
+
+The previous major version is stable for production and will receive patch updates as needed, but there
+will be no new features added. Patch releases are automatically deployed upon merge by GitHub
+Actions.
+
+The current major version is also stable and receives new feature and patch updates. Patch releases are
+automatically deployed upon merge by GitHub Actions, and minor releases are manually deployed at the
+end of each sprint.
+
+The next major version is typically an unstable environment and has major breaking changes. You are
+welcome to pull this version down for local development and experimentation, but we generally
+recommend against using it in production until the first stable version has been released.
+
```

Comment:
is it worth pointing out where this info can be found or where we can see the current version? Example: You can find out current `here` 

Also i know this has come up before but do we host a storybook with a previous version and a current version?

Reply:
Good callouts! I'd like to add another section about our repo branches and where to find things.

We don't have a stable URL reference for previous Storybook versions, but you can find the Chromatic artifacts.

I think I would like that information to live in our development docs in a dedicated section and / or an FAQ.